tox.ini

```diff
@@ -1,13 +1,12 @@
 [tox]
 envlist =
-    py{37,38,39}-django{22,32}
+    py{37,38,39}-django{32}
```

Comment:
In the setup.cfg I saw that we also have python 3.10?

Reply:
uh yeah, that should be added indeed since it's also already listed in the Github CI workflow matrix